specification/maps/data-plane/Search/preview/1.0/search.json

```diff
@@ -1249,6 +1442,8 @@
             "in": "body",
             "description": "This represents the geometry for one or more geographical features (parks, state boundary etc.) to search in and should be a GeoJSON compliant type. Please refer to [RFC 7946](https://tools.ietf.org/html/rfc7946) for details.",
             "required": true,
+            "x-ms-client-flatten": true,
+            "x-ms-client-name": "geometryParameter",
```

Comment:
@johanste your take on `x-ms-client-flatten` for body parameters?

Reply:
This works well but still generates the intermediate model classes (SearchAlongRouteRequestBody, SearchInsideGeometryRequestBody) so the gain is pretty small

Reply:
This caused problem with the TS SDK so I am going to revert it.